model/installation_request_test.go

```diff
@@ -724,3 +821,7 @@ func TestNewAssignInstallationGroupFromReader(t *testing.T) {
 func sToP(s string) *string {
 	return &s
 }
+
+func iToP(i int64) *int64 {
```

Comment:
We need to refactor this functions to a common place, we have them spread all over test files.

Reply:
Yeah, agreed. I will create a ticket to cleanup everything.